src/components/Governance/index.tsx

```diff
@@ -59,6 +61,19 @@ const GovernancePortal = observer(() => {
       <GovernanceFilterDialog open={showGovernanceFilters} onClose={toggleShowDialog} applyFilter={applyFilter} />
 
       <EventsTable events={governancePortal.timelockEvents} filters={filters} />
+
+      <Grid container justifyContent="flex-end" alignItems="center">
+        <Button onClick={() => setShowProposalModal(true)} variant="outlined" color="primary">
+          PROPOSER ROLE
```

Comment:
should be "Propose". the check to show this button on the screen should be TimelockContract.hasRole(PROPOSER_ROLE.addr, user.addr) in essence.